### PR TITLE
Remove notice about redis disk resizing

### PR DESCRIPTION
**Pull request by Kidswiss**

### CODE REVIEW

**Review by akosma — APPROVED** [review]

**Review by lieneluksika — APPROVED** [review]

**Comment by Kidswiss** [comment]
I've added a new note about when service restarts happen, can you have another look if it makes sense? thanks.

File: docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc [hunk 1]
```diff
@@ -96,7 +95,7 @@ spec:
     name: postgres-creds
 ----
 
-IMPORTANT: Changing disk size after initial provisioning is currently not supported.
+NOTE: When changing the disk size from a plan, it will restart the redis pods. Any subsequent disk resizes should not trigger restarts.
```
Comment (lieneluksika) [inline docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc:99]: Does it mean that any other disk size changes after the first one will not trigger restart?

Comment (Kidswiss) [reply docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc:99]: Yes.

If you have suggestions to re-phrase it better, please feel free to suggest :D

Comment (lieneluksika) [reply docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc:99]: thinking..
does should mean that it could? :D

Comment (Kidswiss) [reply docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc:99]: I haven't observed it, but I'm also not 100% sure...

In my tests it never triggered a restart after the first time.

File: docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc [hunk 2]
```diff
@@ -96,7 +95,7 @@ spec:
...
```
Comment (lieneluksika) [inline docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc:98]:
```suggestion
NOTE: Changing the disk size from a plan will restart the Redis pods. Any further disk size changes should not trigger a restart.
```

Comment (Kidswiss) [reply docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc]: I've modified your suggestion slightly, what do you think?

Comment (lieneluksika) [reply docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc]: approved

**Review by lieneluksika — APPROVED** [review]

**Pull request merged**